tools/container-converter/src/image_builder/enclave.rs

```diff
@@ -103,7 +107,9 @@ impl EnclaveSettings {
             user_name: input_image.details.config.user.clone(),
             env_vars: vec![rust_log_env_var("enclave")],
             is_debug: converter_options.debug.unwrap_or(false),
-            enable_overlay_filesystem_persistence: converter_options.enable_overlay_filesystem_persistence.unwrap_or(false)
+            enable_overlay_filesystem_persistence: converter_options.enable_overlay_filesystem_persistence.unwrap_or(false),
+            ccm_backend_url: CcmBackendUrl::new(converter_options.ccm_configuration.clone().unwrap_or_default().ccm_url.as_str()).unwrap_or_default(),
+            dsm_configuration: converter_options.dsm_configuration.clone().unwrap_or_default()
```

Comment:
Nitpick: in Rust you can have a `,` at the end here. Then next time you add a field, no line needs to be removed just because it needs a `,`